app/api/settings.py

```diff
@@ -99,7 +111,15 @@ def slack_schduler():
         ret = mongo.db.schdulers_msg.find({
         })
         ret = [serialize_doc(doc) for doc in ret]
-        return jsonify(ret)
+        default=[{
```

Comment:
why are we defined in 2 times. move this to config and define it one time only

Reply:
you also need to make changes to scheduler based on these default values i dont see it